<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDateTime</class>
 <widget class="QDialog" name="DialogDateTime">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>525</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">修改日期时间</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 28px &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutForm">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frameTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>58</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>58</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frameTitle
{
	background-color: transparent;
	border-image: url(:/img/widget/DialogTitle.png) 0 12 0 12;
	border-left: 12px transparent;
	border-right: 12px transparent;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayoutTitle">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel#labelTitle
{
	color: white;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>系统日期时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="xPushButton" name="pushButtonClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pushButtonClose:!pressed
{
	border-image: url(:/img/widget/BtnCloseNormal.png);
}

QPushButton#pushButtonClose:pressed
{
	border-image: url(:/img/widget/BtnClosePress.png);
}
</string>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frameMainArea">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="IsMainArea" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayoutFrame">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="labelDate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>系统日期：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" alignment="Qt::AlignRight">
       <widget class="xDateEdit" name="dateEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="displayFormat">
         <string notr="true">yyyy-MM-dd</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
        <property name="HasBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="6">
       <widget class="xClock" name="widgetClock" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>250</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2" alignment="Qt::AlignRight">
       <widget class="xTimeEdit" name="timeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="specialValueText">
         <string notr="true"/>
        </property>
        <property name="displayFormat">
         <string notr="true">HH : mm : ss</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
        <property name="HasBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="labelTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>系统时间：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="xPushButton" name="BtnCancel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset resource="xPad.qrc">
          <normaloff>:/img/32X32/Cancel.png</normaloff>:/img/32X32/Cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="IsBoxBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="xPushButton" name="BtnOK">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>70</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset resource="xPad.qrc">
          <normaloff>:/img/32X32/OK.png</normaloff>:/img/32X32/OK.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="IsBoxBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xPushButton</class>
   <extends>QPushButton</extends>
   <header>xpushbutton.h</header>
   <slots>
    <slot>KeyADDClick()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>xClock</class>
   <extends>QWidget</extends>
   <header>xclock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>xDateEdit</class>
   <extends>QDateEdit</extends>
   <header>xdateedit.h</header>
   <slots>
    <signal>clicked(QDateEdit::Section)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>xTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>xtimeedit.h</header>
   <slots>
    <signal>clicked(QTimeEdit::Section)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>timeEdit</tabstop>
 </tabstops>
 <resources>
  <include location="xPad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnOK</sender>
   <signal>clicked()</signal>
   <receiver>DialogDateTime</receiver>
   <slot>SaveExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogDateTime</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogDateTime</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>clicked(QDateEdit::Section)</signal>
   <receiver>DialogDateTime</receiver>
   <slot>dateClicked(QDateEdit::Section)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>625</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEdit</sender>
   <signal>clicked(QTimeEdit::Section)</signal>
   <receiver>DialogDateTime</receiver>
   <slot>timeClicked(QTimeEdit::Section)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SaveExit()</slot>
  <slot>dateClicked(QDateEdit::Section)</slot>
  <slot>timeClicked(QTimeEdit::Section)</slot>
 </slots>
</ui>
